packages/neon/neon/lib/src/blocs/apps.dart

```diff
@@ -164,9 +164,7 @@ class AppsBloc extends InteractiveBloc implements AppsBlocEvents, AppsBlocStates
     unawaited(openNotifications.close());
     unawaited(appVersions.close());
 
-    for (final app in _allAppImplementations) {
-      app.dispose();
-    }
```

Comment:
Why is this removed?

Reply:
The `AppsBloc` is not really the owner of the `appImplementations`.
I added a disposal for them at the provider level in neon.dart.

I don't know why we should ever need the `appImplementations` without the `AppsBloc` (or even without the `AccountsBloc`) but it's now possible. Otherwise the `AppsBloc` could dispose our apps while we still need them elsewhere.